salesagility/SuiteCRM
# Fix Chart Display For Reports With Multiple Fields

## Description
Chart query is now built separately from the main dataset query, so that it only selects the specified fields set for its x and y axis. Everything else remains the same e.g. where builder method so that the datasets are the same, The only differences are the fields selected.


## Motivation and Context
The chart was not displaying correctly when it was presented with more than 2 fields.


## How To Test This
Try a variety of different reports and check the totals add up for both the table dataset and the chart dataset.
Try to display a report where fields are being pulled in from multiple tables.


## Types of changes

### Final checklist
